### PR TITLE
Highlight social elements on borrower profile experiment

### PR DESCRIPTION
**Pull request by christian14b**
This pr adds a List of some lenders below `Help fund this loan section` on Borrower Profile and modifies the `powered by` text showing the total count of lenders as shown in the below image.

Screen Shot 2022-06-30 at 17 46 21

### CODE REVIEW

File: src/components/BorrowerProfile/LendersList.vue [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+		isValidLength(idx) {
+			return (this.isMobile && idx < 2) || (!this.isMobile && idx < 3);
+		},
+		toogleLightBox() {
+			this.$emit('tooglelightbox');
+		}
+	},
+	computed: {
+		isSticky() {
+			return true;
```
Comment (dyersituations) [inline src/components/BorrowerProfile/LendersList.vue:87]: Will this ever be more than `return true`? If not, this computed can be removed.

File: src/components/BorrowerProfile/LendersList.vue [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+			return inviterIndex >= 0 ? [
+				this.lenders[inviterIndex],
+				...this.lenders.slice(0, inviterIndex),
+				...this.lenders.slice(inviterIndex + 1, this.lenders.length - 1)
+			] : this.lenders;
+		},
+		lendersListName() {
+			if (this.lenders.length < 4) return this.sortedLenders[0].name;
+			const filteredLenders = [...this.sortedLenders].slice(0, this.limit);
+			// eslint-disable-next-line max-len
```
Comment (dyersituations) [inline src/components/BorrowerProfile/LendersList.vue:106]: This eslint disable is not needed.

File: src/components/BorrowerProfile/LendersList.vue (hunk 1 above)
Comment (christian14b) [reply src/components/BorrowerProfile/LendersList.vue:87]: Nice catch! I'll remove it

**Review by mcstover — APPROVED** [review]
Let's try this out. Please test it thoroughly in dev!

**Pull request merged**